src/components/Questions/List.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <v-layout row wrap>
+      <v-flex xs6>
+        <h1>Questions</h1>
+      </v-flex>
+
+      <v-flex xs6 class="text-xs-right">
+        <v-btn color="info">
+          Ask Question
+        </v-btn>
+      </v-flex>
+    </v-layout>
+    <div v-if="loading" class="text-xs-center mt-5">
+      <v-progress-circular
+        :width="1"
+        color="red"
+        :size="70"
+        indeterminate
+      />
+    </div>
+
+    <div v-for="question in questions" v-else :key="question.id">
```

Comment:
it's not a good practice to keep `v-for` and `v-else` in same element. Split them in next mode:
```
<template v-else>
    <div v-for="question in questions" :key="question.id">
         // ...
    </div>
</template>
```

Reply:
fixed